languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -1532,13 +1549,15 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         <message>Laŭ ReVo, «\1» estas arkaismo. Uzu <suggestion><match no="1" regexp_match=".*p(.*)" regexp_replace="kolb$1"/></suggestion>.</message>
         <example correction="kolbo">La <marker>pafilkapo</marker> de fusilo.</example>
       </rule>
+      <!-- Not actually an archaism
       <rule>
         <pattern>
           <token inflected="yes">patriarĥo</token>
         </pattern>
         <message>Laŭ ReVo, «\1» estas arkaismo. Uzu <suggestion><match no="1" regexp_match=".*ĥ(.*)" regexp_replace="patriark$1"/></suggestion>.</message>
         <example correction="patriarko">La <marker>patriarĥo</marker> de Romo.</example>
       </rule>
+      -->
```

Comment:
Indeed, patriarĥo is not marked as archaism in ReVo. 
I can't remember why this rule was added then. Perhaps the tag in ReVo was changed.
We can remove the rule instead of commenting it out.

Reply:
I removed the whole rule in the last commit.